stacks/config.py

```diff
@@ -0,0 +1,44 @@
+import os
+import os.path
+import botocore.config
+
+from .utils import load_yaml
+
+AWS_CONFIG_FILE = os.environ['HOME'] + '/.aws/credentials'
+DEFAULT_REGION = 'us-east-1'
```

Comment:
> But maybe things like this should be issues, not changed in this PR.

Yes. I didn't want to make too many changes to code logic in this PR. The purpose of this PR is to restructure code to look like a proper python project.